internal/command/logs/vm.go

```diff
@@ -0,0 +1,38 @@
+package logs
+
+import (
+	"fmt"
+	"github.com/cirruslabs/orchard/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+func newLogsVMCommand() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "vm NAME",
+		Short: "Retrieve VM logs",
+		RunE:  runLogsVM,
+		Args:  cobra.ExactArgs(1),
+	}
+
+	return command
+}
+
+func runLogsVM(cmd *cobra.Command, args []string) error {
+	name := args[0]
+
+	client, err := client.New()
+	if err != nil {
+		return err
+	}
+
+	lines, err := client.VMs().Logs(cmd.Context(), name)
+	if err != nil {
+		return err
+	}
+
+	for _, line := range lines {
+		fmt.Print(line)
```

Comment:
I guess this should be Println or something: currently `orchard logs vm xxx` glues all lines together (also I'm not sure about it, but it looks like there's some synchronization issue with reading the lines):

```
$ orchard create vm v --startup-script 'for i in $(seq 1 100); do echo "$i."; done'
....
$ orchard logs vm v
67.1.68.2.77.11.78.12.79.13.14.80.15.81.82.16.83.17.84.18.19.85.86.20.69.3.87.21.22.88.23.89.90.24.25.91.26.92.27.93.94.28.29.95.30.96.4.70.97.31.98.32.99.33.100.34.35.36.37.38.39.40.71.5.41.42.43.44.45.46.47.48.49.50.6.72.51.52.53.54.55.56.57.58.59.60.73.7.61.62.63.64.65.66.74.8.9.75.76.10.$
```